flashing/mac_converter.py

```diff
@@ -0,0 +1,31 @@
+from pyocd.core.target import Target
```

Comment:
Do you know how stable this operation is if we modify the firmware or update the sdk? Please correct me if I'm wrong, but I have a feeling the raw memory address would likely change, which will make this confusing because it will always output something, even if it's incorrect.

Reply:
As far as I can tell from my experience and the infos I got from the Nordic forums (where I took the inspiration for the code) this is fairly stable. 

It reads the MAC address which is encoded in the hardware by Nordic. I tried this command on a brand new chip which has not been flashed with anything before (at least by me) and it still yields a mac address. It stays the same after flashing the softdevice and firmware.

Reply:
Thanks, that looks good. A caveat is that we allow for configuring a MAC address on the application level (see [here](https://github.com/rbaron/b-parasite/blob/main/code/b-parasite/src/prst/ble.c#L145)). Do you know how that would work? Does this always return the factory-assigned, static MAC?

Reply:
I saw that line in the code (which confused me at first...). I did not try to override the mac address myself so I don't know at the moment. 

My guess is, that the factory MAC address will still be available and will be overridden on application level by the code. But no prove here...

Reply:
From what I can tell this MAC address is the random, static, factory-assigned MAC address (the read-only [`DEVICEADDR` registers](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.nrf52832.ps.v1.1%2Fficr.html)). I would suggest two options:
1. Make it clear that the output MAC address is the `DEVICEADDR` (and not the application override, if it exists)
2. Remove it or move to a separate CLI, as it's orthogonal to flashing (also it seems like the address is the same for both nrf52840 and nrf52833, which we also support)

Reply:
Ok, thanks for digging this up. 

I would rather keep it as a console output statement because it is quite helpful if you are batch flashing devices to keep track which is which. 😉 

We could name it `Static manufacturer MAC address` so it will be clear, that it does not change and is assigned by the chip manufacturer.

Reply:
Sounds good. I'm okay with changing the output to `Static manufacturer MAC address (DEVICEADDR)`. I would also rename `mac_converter.py` to `mac_reader.py`.